### PR TITLE
Fix workflow dispatch ui

### PR DESCRIPTION
**Pull request by TKaxv-7S**
I have fixed the position of this button

image

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -304,7 +304,7 @@ replace github.com/hashicorp/go-version => github.com/6543/go-version v1.3.1
 
 replace github.com/shurcooL/vfsgen => github.com/lunny/vfsgen v0.0.0-20220105142115-2c99e1ffdfa0
 
-replace github.com/nektos/act => gitea.com/pangliang/act v0.251.2-0.20231207083920-176e5f65fa26
```
Comment (pangliang) [inline go.mod:307]: There is no need to modify this to change it back. See the related discussion of [discussion_r1418576123](https://github.com/go-gitea/gitea/pull/28163#discussion_r1418576123). I need to make some changes to the act module. The PR is waiting for them to process, and then I will change it back to the new version of gitea/act.

Comment (TKaxv-7S) [reply go.mod]: Ok, I have removed this commit.

**Comment by pangliang** [comment]
Thanks for your contribution 👍

**Comment by TKaxv-7S** [comment]
> Thanks for your contribution 👍

You're welcome, you've made more contributions. Thank you

**Pull request merged**